openmdao/solvers/nonlinear/nonlinear_block_jac.py

```diff
@@ -41,3 +51,16 @@ def _mpi_print_header(self):
                 header += prefix + pathname + "\n"
                 header += prefix + nchar * "="
                 print(header)
+
+    def _run_apply(self):
+        """
+        Run the apply_nonlinear method on the system.
+        """
+        system = self._system
+
+        # If this is a parallel group, check for analysis errors and reraise.
+        if len(system._subsystems_myproc) != len(system._subsystems_allprocs):
+            with MultiProcFailCheck(self._system.comm):
```

Comment:
forgot to use the 'system' temporary variable here

Reply:
Fixed